blog-posts/scoped-registry.md

```diff
@@ -0,0 +1,100 @@
+---
+path: "/blog/scoped-registry/"
+date: "2021-12-30"
+title: "NPM scoped registry and authentication"
+featuredImage: images/npm-scoped-registry.jpg
+attribution:
+    creator: Paul Esch-Laurent
+    source: https://unsplash.com/photos/oZMUrWFHOB4
+seoMetaText: create scoped packages and scoped registry and configure authentication via npmrc
+teaserText: What are scoped packages, scoped registry and configure authentication via npmrc
+author: Mark Altmann
+authorSummary: dev@satellytes, ❤️ for Wombats
+
+---
+## TL;DR
+
+Packages in npm can be grouped together with scopes. Scopes are preceded by a `@` followed by the scope name and the package name separated by `/` e.g. `@scope/package`. To set a specific registry for a scope you can use scoped registries. To set a scoped registry open up the .npmrc file and enter the following:
+
+`@scope:registry=https://<your registry>` where `@scope` is your scope name and `<your registry>` is the registry where your scoped packages are hosted. 
```

Comment:
To make it more readable I would put this into a full-width code block (three ` ticks). This way you can see on first glance what the file should look like:

```
@scope:registry=https://<your registry>
```

Reply:
good idea. Thanky :+1: